flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointTestBase.java

```diff
@@ -123,6 +126,20 @@
 
     @Rule public TestName name = new TestName();
 
+    @BeforeClass
+    public static void beforeAll() {
+        // set to some high enough number, it is recommended to have as many arenas as slots
+        // If a single buffer pool is shared between all tms we should have tms * slots_per_tm
+        // This should be the maximum across all tests run
+        SharedPoolNettyShuffleServiceFactory.bufferPool = new NettyBufferPool(60);
```

Comment:
can you encapsulate modifications of this field? At the very least via some static methods in `SharedPoolNettyShuffleServiceFactory`?